README.md

```diff
@@ -149,7 +149,7 @@ The plugin provides props for extra customization. Every time you change the pro
 ```javascript
 import WifiManager from "react-native-wifi-reborn";
 
-WifiManager.connectToProtectedWifiSSID(ssid, password, isWep).then(
```

Comment:
this example here even hints that `connectToProtectedWifiSSID` is the one we should use. However, the example is incorrect in that the parameters need to be an object.